tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -125,16 +127,10 @@ public void RunAllTests()
             Assert.IsFalse(hasFailed, $"Test Failed. {stringBuilderMessage}");
         }
 
-        private static void LogMessage(string message, StringBuilder stringBuilderMessage)
-        {
-            Console.WriteLine(message);
-            stringBuilderMessage.AppendLine(message);
-        }
 
         /// <summary>
         /// Runs through the standard acquire token flow, using the default app configured UiBehavior = Login
         /// </summary>
-        [Test]
```

Comment:
I don't see any value in having these as tests and having an overarching RunAllTests test. It introduces bugs, like the one I just fixed.

Reply:
The intent was to allow us the option to run tests individually so that we can acquire screenshots of failed tests. The problem with having all tests run at once is you lose the test results of individual tests when they fail (screen shots).

Reply:
Ok, I will revert this.